packages/ilib-loctool-webos-json/test/JsonFileAppinfo.test.js

```diff
@@ -837,4 +837,88 @@ describe("jsonfile", function() {
 
         expect(ajf.getLocalizedPath("fr-FR")).toBe("localized_json/fr/");
     });
-});
\ No newline at end of file
+    test("JsonLocalzeTextWithBaseTranslations2", function() {
+        expect.assertions(2);
+        var ajf = new JsonFile({
+            project: p,
+            type: ajft
+        });
+        expect(ajf).toBeTruthy();
+        ajf.parse({
+            "id": "app",
```

Comment:
This is supposed to be a string with json in it, right?

Reply:
I've updated it with string format. If my changes are not what you intended, please let me know.